templates/managed-rds-postgresql-v1.j2

```diff
@@ -0,0 +1,252 @@
+    Description: Multi-AZ database
+    Type: String
+    Default: 'true'
+    AllowedValues:
+      - 'true'
+      - 'false'
+    ConstraintDescription: must be true or false.
+Conditions:
+  HasMksKeyId: !Not [!Equals ["", !Ref KmsKeyId]]
+Resources:
+  {% if sceptre_user_data.IngressWhitelist is defined %}
+  DBInstanceSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: "Open database on defined port for access by whitelisted IP"
+      VpcId: !ImportValue 'Fn::Sub': '${AWS::Region}-${VpcName}-VPCId'
+      SecurityGroupIngress:
+      {% for CIDRBlock in sceptre_user_data.IngressWhitelist %}
+        - IpProtocol: tcp
+          FromPort: !Ref 'DBPortNumber'
+          ToPort: !Ref 'DBPortNumber'
+          CidrIp: {{ CIDRBlock }}
+      {% endfor %}
+      SecurityGroupEgress:
+        - CidrIp: "0.0.0.0/0"
+        FromPort: -1
+        ToPort: -1
+        IpProtocol: "-1"
+      Tags:
+        - Key: "Department"
+          Value: !Ref Department
+        - Key: "Project"
+          Value: !Ref Project
+        - Key: "OwnerEmail"
+          Value: !Ref OwnerEmail
+  {% endif %}
+  DBInstance:
+    Type: 'AWS::RDS::DBInstance'
+    Metadata:
+      'AWS::CloudFormation::Init':
+        configSets:
+          SetupCfn:
+            - cfn_hup_service
+          SetupVolume:
+            - tag_volume
+        # Cfn-hup setting, it is to monitor the change of metadata.
+        # When there is change in the contents of json file in the metadata section, cfn-hup will call cfn-init.
+        cfn_hup_service:
+          files:
+            /etc/cfn/cfn-hup.conf:
+              content: !Sub |
+                [main]
+                stack=${AWS::StackId}
+                region=${AWS::Region}
+                verbose=true
+                interval=5
+              mode: "000400"
+              owner: root
+              group: root
+            /etc/cfn/hooks.d/cfn-auto-reloader.conf:
+              content: !Sub |
+                [cfn-auto-reloader-hook]
+                triggers=post.update
+                path=Resources.DBInstance.Metadata.AWS::CloudFormation::Init
+                action=/opt/aws/bin/cfn-init -v --stack ${AWS::StackName} --resource DBInstance --configsets SetupCfn,SetupVolume --region ${AWS::Region}
+              mode: "000400"
+              owner: root
+              group: root
+        /lib/systemd/system/cfn-hup.service:
+          content: !Sub |
+            [Unit]
+        Description=cfn-hup daemon
+
+        [Service]
+        Type=simple
+        ExecStart=/opt/aws/bin/cfn-hup
+        Restart=always
+
+        [Install]
+        WantedBy=multi-user.target
+        mode: "000400"
+        owner: root
+        group: root
+    commands:
+      01_enable_cfn-hup:
+        command: "/bin/systemctl enable cfn-hup.service"
+      02_start_cfn-hup:
+        command: "/bin/systemctl start cfn-hup.service"
+        tag_volume:
+      commands:
+        01_tag_root_disk:
```

Comment:
We tag root volume in the provisioner because we allow users to set the root volume size so we want to track the cost.  When provisioning an RDS the root volume is most likely very small and most likely ephermeral so there is no additional cost.  My suggestion is to just remove this entire `AWS::CloudFormation::Init` and don't worry about tagging the root volume.